data/presets/presets/indoor/elevator.json

```diff
@@ -0,0 +1,20 @@
+{
+    "icon":"temaki-elevator",
+    "fields": [
+        "level",
+        "name"
```

Comment:
I'd try `ref` instead of `name` for these since they're more likely to have reference codes than real names.